### DISCUSSION TITLE
Support ISO GQL:2024 : avoid vendor lock-in, facilitate benchmarking/comparability

### DISCUSSION THREAD

**Discussion by annainfo** [terminusdb/terminusdb#2088] [Ideas]
Implement GQL:2024 (FDIS stage, ETA 2024-04/2024-05) or at least the SELECT/MATCH part of SQL:2023 Part 16 SQL/PGQ which shares the GPM.

1) 2019-09-10 ISO GQL - stage 10.99 New project approved https://www.iso.org/standard/76120.html
2) [pre 2022-03-30, couldn't find a date] "A Bit later" at https://github.com/orgs/terminusdb/projects/2
3) 2023-01-17 (a year and two weeks ago) 
3.1) "Support of industry standard QL/GPM - openCypher - GQL" - including info about stages of GQL, SQL/PGQ and the shared GPM https://github.com/orgs/terminusdb/discussions/1657 
3.2) "GPM is stable - any news regarding GQL support?" including answer by a maintainer: no capacity, and "We think we need to build a combinator library in rust." https://github.com/orgs/terminusdb/discussions/1662 
4) 2023-06-01 SQL:2023 published including "ISO/IEC 9075-16:2023, Information technology, Database languages SQL, Part 16: Property Graph Queries (SQL/PGQ)" https://www.iso.org/standard/79473.html
5) 2023-06-20 message by maintainer "We don't have active plans to add SPARQL support, but it would not be a particularly hard project to do so." https://github.com/orgs/terminusdb/discussions/830
6) 2023-10-16 "WOQL - rewrite in order" - no mention of GQL https://github.com/orgs/terminusdb/discussions/1996 
7) 2024-01-16 "The publication process takes six to eight weeks so ISO/IEC 39075:2024 Information technology — Database languages — GQL should officially be published in late April or early May of 2024!" https://www.gqlstandards.org/
8) 2024-01-26 GQL:2024 at stage 50:20 https://www.iso.org/standard/76120.html

**Comment by alexander-mart** [comment]
I suggest deleting this duplicate topic and continuing in the [previously started thread](https://github.com/orgs/terminusdb/discussions/1657)

Backuped text is there:
https://github.com/orgs/terminusdb/discussions/1657#discussioncomment-12737361
